src/azure-cli/HISTORY.rst

```diff
@@ -180,6 +180,11 @@ Release History
 **ARM**
 
 * Fix `az resource tag`: Recovery Services Vault tags cannot be updated
+* Added new command groups
```

Comment:
Please sync your PR with dev branch and make your release note in the right position.

Reply:
fixed.